test/unit/specs/store/delegation.spec.js

```diff
@@ -3,11 +3,12 @@ import setup from "../../helpers/vuex-setup"
 let instance = setup()
 
 describe("Module: Delegations", () => {
-  let store
+  let store, node
```

Comment:
It's better form to use separate lines for each variable:

```JavaScript
let store
let node
```

Reply:
To be consistent, I would need to change this everywhere. Let's not do it until we come up for a rule about it, ok?